src/Toast/index.jsx

```diff
@@ -78,7 +78,7 @@ Toast.defaultProps = {
 
 Toast.propTypes = {
   /** A string or an element that is rendered inside the main body of the `Toast`. */
-  children: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
```

Comment:
I think you can just do this, which covers strings, elements, and anything else valid:
```suggestion
  children: PropTypes.node.isRequired,
```